src/main/java/util/TextUI.java

```diff
@@ -81,17 +101,30 @@ public static void displayList(List<String> options, String msg){
         }
     }
 
+    public static void displayListOfProducts(List<Product> options, String msg){
+        displayMsg("");
+        displayMsg(msg);
+        displayMsg("");
+
+        int i = 1;
+
+        for (Product option : options) {
+            displayMsg(i+": "+option);
+            i++;
+        }
```

Comment:
skulle det ikke havde været et almindeligt for loop

Reply:
Kan man også, men det der fungere